patches/@credo-ts__openid4vc@0.5.11.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/build/shared/issuerMetadataUtils.js b/build/shared/issuerMetadataUtils.js
```

Comment:
do we want to keep this patch? Or do it in thefork?

Reply:
as it's related to the removal of mdoc, i think we can keep it here for now (as the correct code is already in another branch). If it causes issues I'll move it